actors/builtin/miner/deadline_state.go

```diff
@@ -440,6 +441,11 @@ func (dl *Deadline) PopEarlyTerminations(store adt.Store, max uint64) (earlyTerm
 		return nil, false, xerrors.Errorf("failed to walk early terminations bitfield for deadlines: %w", err)
 	}
 
+	// Removed finished partitions from the index.
+	for _, finished := range partitionsFinished {
+		dl.EarlyTerminations.Unset(finished)
+	}
```

Comment:
We can do this for consistency (and I guess it's easier to reason about), but it's shouldn't be necessary (technically).

Reply:
For me it's actually the opposite, but I guess that's because I've worked on the bitfield code but not the AMT code :laughing:.

I agree that maintaining the invariant could be tricky if we ever decide to perform eager updates, or anything like that.